actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -687,6 +683,18 @@ def time_tag(date_or_time, *args, &block)
 
         content_tag("time".freeze, content, options.reverse_merge(datetime: datetime), &block)
       end
+
+      private
+
+        def normalize_distance_of_time_argument_to_time(value)
+          if value.is_a?(Numeric)
+            Time.at(value)
+          elsif value.respond_to?(:to_time)
+            value.to_time
+          else
+            raise ArgumentError, "#{value} must be convertible to_time"
```

Comment:
One last change, I promise 😄

Using `#{value}` with a `nil` value isn't helpful, e.g:

```
>> value = nil
>> "#{value} can't be converted to a Time value"
=> " can't be coerced to a Time value"
```

It's better to use `inspect`:

```
>> value = nil
>> "#{value.inspect} can't be converted to a Time value"
=> "nil can't be converted to a Time value"
```

Can you also tweak the message to what I've used in this comment which is more in line with other similar Ruby messages, thanks.